### ISSUE TITLE
Pagination "skip to last page" is not selected after click

### ISSUE THREAD

**Issue by fenta23** [swimlane/angular2-data-table#324]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
If you jump to the last page via pagination, instead of the last element the pre last element get selected.

**Expected behavior**
Pagination shows last item selected

**Reproduction of the problem**
Go to the Demo page: http://swimlane.github.io/angular2-data-table/#
chose a Demo with pagination (100k rows, actually there are only 10k, but doesn't matter;))
Click on the skip to last page icon in pagination. Prelast get selected.

**What is the motivation / use case for changing the behavior?**
Obvious, the last element get selected.

**Please tell us about your environment:**
Windows 10
Node v6.9.1

* **Table version:** 0.7.x
latest

* **Angular version:** 2.0.x
'@angular/core": "2.0.1",

* **Browser:** 
Chrome Version 54.0.2840.99 m (64-bit)
Firefox 49.0.2

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
typescript :~2.0.2

**Comment by fenta23** [comment]
It works in iOS10.1.1 Safari and IE Edge as expected.

**Comment by fenta23** [comment]
It is somehow related to the height of the table itself. if you set your demo 100k table to 500px, it works. I will have a look later in your calculation. Can you give me a hint where to look?

**Comment by amcdnl** [comment]
Fixed in 2.0 ( i think lol )
